bench/wtperf/wtperf.c

```diff
@@ -2361,6 +2361,7 @@ start_run(WTPERF *wtperf)
         total_ops = wtperf->insert_ops + wtperf->modify_ops + wtperf->read_ops + wtperf->update_ops;
 
         run_time = opts->run_time == 0 ? 1 : opts->run_time;
+        printf("Run time set to: %u", run_time);
```

Comment:
Is this a debug printf? If we're keeping it could you change to `lprintf` like the surrounding logs?